firebase-vscode/src/logger-wrapper.ts

```diff
@@ -0,0 +1,13 @@
+
+import { logger as cliLogger } from "../../src/logger";
+
+export const pluginLogger: Record<string, any> = {};
```

Comment:
I think this any can be LogMethod instead (exported from Logger), or at least something like  `(...args) => void`

Reply:
Tried using LeveledLogMessage but for some reason Typescript gets confused and keeps thinking the return type needs to be the winston Logger and not the Logger interface in src/logger.ts. Went with (...args) => void.